recipes/gnss-sdr/meta.yaml

```diff
@@ -0,0 +1,142 @@
+{% set version = "0.0.18" %}
+
+package:
+  name: gnss-sdr-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/gnss-sdr/gnss-sdr/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d5533370de188d50c214a5bb7832a433ceb09369c3e3296a4fa34fa1aefb685e
+  patches:
+    - 0001-Manually-specify-compiled-in-GNSSSDR_INSTALL_DIR.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+outputs:
+  - name: volk-gnss-sdr
```

Comment:
Each output with libraries needs run_exports to ensure ABI compatability downstream.

Reply:
Done!